WcaOnRails/app/assets/javascripts/competitions.js

```diff
@@ -34,6 +34,7 @@ onPage('competitions#edit, competitions#update, competitions#admin_edit, competi
 
   $('input[name="competition[event_restrictions]"]').on('change', function() {
     $('.competition_event_restrictions_reason').toggle(this.checked);
+    $('.competition_events_per_registration_limit').toggle(this.checked);
```

Comment:
I would love not to introduce new jQuery for this, because we want to get rid of jQuery. 
What about replacing it with a new JS approach using Query Selector?

Reply:
Updated.